src/pages/[platform]/deploy-and-host/fullstack-branching/pr-previews/index.mdx

```diff
@@ -44,4 +40,35 @@ With fullstack previews, you can set up ephemeral fullstack environments on ever
 1. The pull request preview is deployed on Amplify and available at `pr-1.appid.amplifyapp.com`.
 1. Once the pull request is merged into `main`, the request is closed and the fullstack environment is also automatically torn down.
 
-Once this feature is available, we will update this page with instructions for generating fullstack previews so you can view and validate proposed changes to your fullstack app before merging pull requests.
+### Prerequisites
+
+Before you get started, make sure you have the following:
+
+- [An Amplify Gen 2 app created](https://docs.amplify.aws/gen2/start/quickstart/)
+- The app should have a `main` (production) branch and a `dev` (feature) branch.
```

Comment:
When testing the below flow of events, I ran into an issue where the CFN stack failed (during `pipeline-deploy`) to create a particular resource since the `dev` branch did not exist:
- Create an app with a `main` branch
- Create a `dev` branch off the `main` branch in the git repo (do not connect this branch in the console)
- Update the `dev` branch and create a PR against `main`
- The deployment in the console for `pr-1` branch fails.

Thus, its necessary to have both a `main` and `dev` branch deployed in the console.

Reply:
The Hosting team is working on rolling out a fix for this issue.